src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+/**
+ * The Duke program implements an application that
+ * allows users to store Tasks in a list, which will
+ * be saved in between sessions.
+ */
+public class Duke {
+
+    private final Ui ui;
+    private final Storage storage;
+    private final Parser parser;
+    private final TaskList taskList;
+
+    /**
+     * Class constructor.
+     *
+     * @param ui       handles interactions with user.
+     * @param storage  handles loading and saving of Tasks.
+     * @param parser   handles user commands.
```

Comment:
inconsistent spacing in the method header comments. I think I saw this in other classes as well so please fix them when u can! :)